adoc/Harbor-Private-Registry-Administration.adoc

```diff
@@ -0,0 +1,348 @@
+:experimental:
+
+[#administration]
+== Administration
+
+=== Setup Administration Access
+
+{spr} User Interface can be accessed from a supported web browser at the location provided as `<ingress_url>` during the installation.
```

Comment:
<ingress_url> was actually replaced by <core_fqdn> at the start of installation section...